crates/fluvio-extension-common/Cargo.toml

```diff
@@ -27,5 +27,5 @@ futures-lite = { version = "1.7.0" }
 thiserror = "1.0.20"
 semver = { version = "1.0.0", features = ["serde"] }
 
-fluvio = { version = "0.11.0", path = "../fluvio", optional = true }
+fluvio = { version = "0.11.1", path = "../fluvio", optional = true }
```

Comment:
Right now the CI, is failing because of this, I think that this change is needed if we are updating fluvio to 0.12.0, but since fluvio is still 0.11.X, we dont need to update the dependency here

Reply:
After talking with Nick C, the changes to the ConsumerConfig were undone, making a version change unnecessary as no changes are made to fluvio crate now.